server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -263,6 +264,15 @@ public static TransportVersion fromString(String str) {
         return TransportVersion.fromId(Integer.parseInt(str));
     }
 
+    // finds the pluggable current version, or falls back to the locally defined current
+    private static TransportVersion findCurrent(TransportVersion fallback) {
+        var versionExtension = VersionExtension.load();
+        if (versionExtension == null) {
+            return fallback;
+        }
+        return new TransportVersion(versionExtension.getCurrentTransportVersionId());
```

Comment:
The comment for this method is not quite right? the fallback is to the passed in fallback, which I guess will be the CURRENT.

Reply:
I reworded the comment.